kvrocks2redis test keep failing in ArchLinux CI

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/kvrocks/issues) and found no similar issues.


### Version

unstable

### Minimal reproduce step

https://github.com/apache/kvrocks/actions/runs/11145141492/job/31005011435

### What did you expect to see?

-

### What did you see instead?

-

### Anything Else?

_No response_

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

```
AssertionError: Data mismatch for key 'set_key_0': source data: '['set_value_0_1', 'set_value_0_2', 'set_value_0_3']' destination data: '['set_value_0_3', 'set_value_0_2', 'set_value_0_1']'
```
I think it is possible that a behavior change in a new version of Redis caused this reordering.

--- a/.github/workflows/kvrocks.yaml
+++ b/.github/workflows/kvrocks.yaml
@@ -540,6 +540,8 @@ jobs:
         run: apt install -y python3-redis
 
       - name: Run kvrocks2redis Test
+        # FIXME: https://github.com/apache/kvrocks/issues/2574
+        if: ${{ !startsWith(matrix.image, 'archlinux') }}
         shell: bash
         run: |
           $HOME/local/bin/redis-server --daemonize yes

